docs/languages/python.md

```diff
@@ -0,0 +1,83 @@
+---
+Order: 8
+Area: languages
+TOCTitle: Python
+ContentId: c2cb770d-571d-4edf-9eb9-b5b8977c21a0
+PageTitle: Python with Visual Studio Code
+DateApproved: 6/2/2016
+MetaDescription: Learn about Visual Studio Code features (syntax highlighting, snippets, linting) for Python development.
```

Comment:
I agree. For SEO I would change IntelliSense to code completion. From  my keyword research code completion is used much more than IntelliSense and it speaks to our net new audience.